tests/monotouch-test/CoreGraphics/PathTest.cs

```diff
@@ -275,8 +275,9 @@ public void IncreaseRetainCountMakeMutable ()
 				using (var copy = p1.Copy ())
 				{
 					var newRetainCount = CFGetRetainCount (copy.Handle);
-					Assert.AreNotEqual (count, newRetainCount, $"Count is {newRetainCount} and was {count}");
-					Assert.IsTrue (count < newRetainCount);
+					Assert.AreEqual (count, newRetainCount);
```

Comment:
whenever there's more than one assert (or better always) add a text message with the assert (stack traces are not always available and can be wrong if something else is broken)